aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -69,6 +69,10 @@ extern "C" void sorgqr_(int *m, int *n, int *k, float *a, int *lda, float *tau,
 extern "C" void dsyev_(char *jobz, char *uplo, int *n, double *a, int *lda, double *w, double *work, int *lwork, int *info);
 extern "C" void ssyev_(char *jobz, char *uplo, int *n, float *a, int *lda, float *w, float *work, int *lwork, int *info);
 
+// geev
+extern "C" void dgeev_(char *jobvl, char *jobvr, int *n, double *a, int *lda, double *wr, double *wi, double* vl, int *ldvl, double *vr, int *ldvr, double *work, int *lwork, int *info);
+extern "C" void sgeev_(char *jobvl, char *jobvr, int *n, float *a, int *lda, float *wr, float *wi, float* vl, int *ldvl, float *vr, int *ldvr, float *work, int *lwork, int *info);
+
```

Comment:
Maybe it also makes sense to enable the complex interface?

Reply:
I can do that, maybe should be a followup PR.